hier_config/__init__.py

```diff
@@ -90,9 +90,12 @@ def __eq__(self, other):
         for self_child, other_child in zip(sorted(self.children), sorted(other.children)):
             if self_child != other_child:
                 return False
-
         return True
 
+    def __ne__(self, other):
```

Comment:
Since this class inherits `HConfigChild` and `__ne__` is defined there, is there a need to define `__ne__` here?